test/integration/framework/etcd.go

```diff
@@ -43,7 +44,14 @@ You can use 'hack/install-etcd.sh' to install a copy in third_party/.
 
 // getEtcdPath returns a path to an etcd executable.
 func getEtcdPath() (string, error) {
-	bazelPath := filepath.Join(os.Getenv("RUNFILES_DIR"), "com_coreos_etcd/etcd")
+	bazelPath := ""
+	switch runtime.GOARCH {
+	case "arm64":
+		bazelPath = filepath.Join(os.Getenv("RUNFILES_DIR"), "com_coreos_etcd_arm64/etcd")
```

Comment:
then you won't need the switch here; instead you could just do
```golang
bazelPath = filepath.Join(os.Getenv("RUNFILES_DIR"), fmt.Sprintf("com_coreos_etcd_%s", runtime.GOARCH), "etcd")
```

Reply:
Done.